polling_stations/apps/data_finder/helpers/routing.py

```diff
@@ -80,6 +82,19 @@ def addresses_have_single_station(self):
             return bool(list(self.polling_stations)[0])
         return False
 
+    def lookup_elections(self):
```

Comment:
This method (which sets `self.elections_response`) is only called in the `route_type` cached property.
Which works for how it is then patched into polling_stations/apps/api/postcode.py, but could be confusing.

Would it make sense to do something like: 

```python
def __init__(self, postcode):
     ...
    self._elections_response = None
    ...

@property
def elections_response(self):
    if not self._elections_response:
        self.lookup_elections()
    return self._elections_response
```